docs/reference/filters.md

```diff
@@ -1343,6 +1343,35 @@ Examples:
 oauthTokeninfoAllKV("k1", "v1", "k2", "v2")
 ```
 
+#### oauthTokeninfoValidate
+
+> This filter is experimental and may change in the future, please see tests for example usage.
+
+The filter obtains token info and allows request if there was no error
```

Comment:
We used tokeninfo as tool name, so it might make sense to stay with tokeninfo here and not use "token info".

Reply:
Its an info about token which it obtains. We also use "token info" in a few other places.